src/main/java/com/main/controller/PersonController.java

```diff
@@ -1,19 +1,26 @@
 package com.main.controller;
 
-import java.util.List;
-
-import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RestController;
 
-import com.main.entity.Message;
+import com.main.dto.ResponseDto;
+import com.main.service.PersonService;
 
 @RestController
 public class PersonController {
-	
-	
-	@GetMapping
-	public List<Message> MessagesByPerson(){
-		return null;
+	@Autowired
+	private PersonService personService;
+
+	@RequestMapping(value = { "/search", "/search/{word}" }, method = RequestMethod.GET)
```

Comment:
Filtering/Search is not recommended b using PathVariables, what other option can you use to Filter/Search?